Controllers/NotesController.cs

```diff
@@ -0,0 +1,55 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Taskmanager.Controllers.InputModels;
+using Taskmanager.Controllers.ViewModels;
+using Taskmanager.Data.Entities;
+using Taskmanager.Services.Interfaces;
+
+namespace Taskmanager.Controllers
+{
+    [ApiController]
+    [Route("api/notes")]
+    public class NotesController : ControllerBase
+    {
+        private readonly INoteService _noteService;
+
+        public NotesController(INoteService noteService)
+        {
+            _noteService = noteService;
+        }
+
+        [HttpGet("{todoItemId}")]
+        public async Task<ActionResult<NoteViewModel>> GetByIdAsync([FromRoute] int todoItemId)
+        {
+            Note note = await _noteService.GetByIdAsync(todoItemId);
```

Comment:
Method name is incorrect as GetById assumes note's id by naming but you pass todoItemId